reference/catalog.v3.yml

```
@@ -21692,7 +21692,7 @@ components:
     Url:
       type: object
       properties:
-        path:
+        url:
```

```suggestion
        path:
```
revert for read-only file